conversationgames/conversationgames.py

```diff
@@ -717,7 +718,7 @@ async def wouldyourather(self, ctx):
         embed = discord.Embed()
         embed.title = "Would you rather.."
         embed.description = strings[i]
-        await ctx.send(embed=embed)
+        await ctx.send(embed=randomize_colour(embed))
```

Comment:
I think the randomize_colour shouldn't be used, can you remove that from the PR?